src/HPWH.cc

```diff
@@ -1323,21 +1041,16 @@ int HPWH::setSetpoint(double newSetpoint, UNITS units /*=UNITS_C*/)
     }
     else
     {
-        if (hpwhVerbosity >= VRB_reluctant)
-        {
-            msg("Incorrect unit specification for setSetpoint.  \n");
-        }
+        send_warning(fmt::format("Incorrect unit specification for setSetpoint.", 0));
```

Comment:
This works for now, but we should probably figure out a way to better handle this with courier. A way to send errors without throwing, so we can accumulate error messages before throwing.

Reply:
This line also doesn't need `fmt::format`.

Reply:
Since these are all returning HPWH_ABORT anyway, maybe they should all turn into `send_error`?